src/main/java/namor/template/calculator/RPNcalculator.java

```diff
@@ -1,24 +1,28 @@
 package namor.template.calculator;
 
-import namor.template.stack.ArrayStack;
+import namor.template.stack.Stack;
 
 public class RPNcalculator {
 
+    private Stack numbers;
 
-    public int calculate(String arithmeticExpression) {
-        ArrayStack numbers = new ArrayStack();
+    public RPNcalculator(Stack stk) {
+        this.numbers = stk;
```

Comment:
Usually this names are the same. No need to make different name. And numbers maybe is not the best name. It doesn't say anything about the purpose of the stack